server/src/modules/session/util.service.ts

```diff
@@ -368,8 +368,8 @@ export class SessionUtilService {
   async #onboardingFlags(user: User) {
     let isFirstUserOnboardingCompleted = true;
     let isOnboardingCompleted = true;
-    const isOnboardingQuestionsEnabled =
-      this.configService.get<string>('ENABLE_ONBOARDING_QUESTIONS_FOR_ALL_SIGN_UPS') === 'true';
+    // const isOnboardingQuestionsEnabled =
```

Comment:
we have to keep this, since we are using cloud same as EE behaviour

Reply:
Sure. we can take this up when LTS cloud is testing.